packages/field-base/test/field-mixin.test.js

```diff
@@ -727,6 +727,119 @@ const runTests = (defineHelper, baseMixin) => {
       });
     });
 
+    describe('accessible-name', () => {
+      beforeEach(async () => {
+        element = fixtureSync(`<${tag} label="Label" helper-text="Helper" error-message="Error Message"></${tag}>`);
+        await nextRender();
+        input = element.querySelector('[slot=input]');
+      });
+
+      it('should not define aria-label if no accessible-name is defined', () => {
+        expect(input.hasAttribute('aria-label')).to.be.false;
+      });
+
+      it('should not change aria-labelledby if no accessible-name is defined', () => {
+        expect(input.hasAttribute('aria-labelledby')).to.be.true;
+      });
+
+      it('should set aria-label when accessible-name is defined', async () => {
+        element.accessibleName = 'accessible name';
+        await nextFrame();
+        expect(input.getAttribute('aria-label')).to.be.equal('accessible name');
+      });
+
+      it('should remove aria-labelledby by if no accessible name is defined', async () => {
+        element.accessibleName = 'acessible name';
+        await nextFrame();
+        expect(input.hasAttribute('aria-labelledby')).to.be.false;
+      });
+
+      it('should remove aria-label when accessible name is cleared', async () => {
+        element.accessibleName = 'accessible name';
+        await nextFrame();
+        element.accessibleName = null;
+        await nextFrame();
+        expect(input.hasAttribute('aria-label')).to.be.false;
+      });
+
+      it('should restore aria-labelledby value if acessible-name is cleared', async () => {
+        const ariaLabelledby = input.getAttribute('aria-labelledby');
+        element.accessibleName = 'acessible name';
+        await nextFrame();
+        element.accessibleName = null;
+        await nextFrame();
+        expect(input.getAttribute('aria-labelledby')).to.be.equal(ariaLabelledby);
+      });
+    });
+
+    describe('acessible-name-ref', () => {
+      beforeEach(async () => {
+        element = fixtureSync(`<${tag} label="Label" helper-text="Helper" error-message="Error Message"></${tag}>`);
+        await nextRender();
+        input = element.querySelector('[slot=input]');
+      });
+
+      /*
+        it('should not override aria-labelledby if no accessible-name-ref is defined', () => {
```

Comment:
Are we waiting for something to complete these tests?

Reply:
No. I forgot to delete it.